__PROJECT NAME__/fastlane/.env

```diff
@@ -0,0 +1,44 @@
+################
+## PROJECT
+################
+PROJECT_NAME="__PROJECT NAME__"
+XCWORKSPACE="${PROJECT_NAME}.xcworkspace"
+XCODE_PROJ="${PROJECT_NAME}.xcodeproj"
+
+################
+## APPSTORE
+################
+TEAM_ID="__TEAM_ID__"
+APPLE_ID="__APPLE_ID__"
+
+################
+## CRASHLYTICS
+################
+CRASHLYTICS_API_TOKEN="__CRASHLYTICS_API_TOKEN__"
+CRASHLYTICS_BUILD_SECRET="__CRASHLYTICS_BUILD_SECRET__"
+
+################
+## APP BUNDLE
+################
+BUNDLE_ID="--BUNDLE-IDENTIFIER--"
+BUNDLE_ID_STAGING="${BUNDLE_ID}.staging"
+BUNDLE_ID_DEBUG="${BUNDLE_ID}.debug"
+
+SCHEME_NAME="__PROJECT NAME__"
+SCHEME_NAME_STAGING="${SCHEME_NAME} staging"
+SCHEME_NAME_DEBUG="${SCHEME_NAME} debug"
+
+################
+## SLACK
+################
+# SLACK_URL=
+# SLACK_USERNAME=
+# SLACK_ICON_URL=
```

Comment:
let's leave some comment here and don't forget to update it in our `README.md` ✌🏻